package-pipeline.ps1

```diff
@@ -23,6 +23,7 @@ Get-ChildItem -Path .\target\* -Include 'azure*' -Exclude '*shaded.jar','*tests.
 StopOnFailedExecution
 copy-item ./worker.config.json pkg
 copy-item ./tools/AzureFunctionsJavaWorker.nuspec pkg/
+Copy-Item ".\annotationLib" pkg\annotationLib -Recurse
```

Comment:
Can we please add / instead of \?

Reply:
I am following v3 style on this line, but I can change it to / for sure

Reply:
Thank you Kaibo, yes please let's have it as /